src/Application/Cli/PreviewCommand.php

```diff
@@ -90,6 +90,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
             sleep(1);
         }
 
+        $output->writeln('<error>HTTP server has stopped working</error>');
+
         return 0;
```

Comment:
If the HTTP server fails, should we return a non-zero exit code?

And should we just simply throw an exception instead of writing to the output? (is the output of the HTTP server silenced or not? I don't remember, I don't want to flood the output uselessly)